### PR TITLE
moneymaker->main fixed

### PR DESCRIPTION
**Pull request by perrymant**
- Moneymaker.java -> Main.java
the argument "-log" should be provided, but will also deal with no/erroneous argument(s).

### CODE REVIEW

File: src/main/java/io/github/perrymant/moneymaker/Application.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    void start(String[] args) {
+        if(args.length == 0) {
+            System.out.println("No arguments given. Proper usage is:\n$ java moneymaker -log");
+            System.exit(0);
+        } else if (args[0].equals("-log")) {
+            new Logger().info(budget.report());
+        } else {
+            System.out.println("Incorrect argument given. Proper usage is:\n$ java moneymaker -log");
+            System.exit(0);
```
Comment (kinbiko) [inline src/main/java/io/github/perrymant/moneymaker/Application.java]: This is done by default when the main method exits. Unfortunately, applications that get run incorrectly tend to return an error code, which is *anything but 0*.

**Review by kinbiko — COMMENTED** [review]
I would hold off on this work until the design has been agreed upon. Once that's done we can working on the implementation - but in reality that'll be next sprint.

**Comment by perrymant** [comment]
Ok, I thought I was being clever :) 
I've simplified it to the essential - is that more in line with what should happen with this branch?

File: src/main/java/io/github/perrymant/moneymaker/Application.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package io.github.perrymant.moneymaker;
+
+class Application {
+    private final Budget budget = new Budget(new Balance(), new TransactionMaker().getTransactions());
+
+    void start(String[] args) {
+        if (args[0].equals("-log")) {
```
Comment (kinbiko) [inline src/main/java/io/github/perrymant/moneymaker/Application.java]: don't even need this check right now.

**Review by kinbiko — APPROVED** [review]
Just simplify that last bit and you may merge without asking for a re-review

**Pull request merged**